### DISCUSSION TITLE
Removing the "FULL" mask in grid_stat

### DISCUSSION THREAD

**Discussion by CPKalb** [dtcenter/METplus#1410] [Statistical Computation] [answered]
Hi:

Is there a way to turn off output for the FULL domain in METplus and only use a poly domain?  I've tried in my file both of these:

GRID_STAT_MASK_POLY = /home/nowcast/tina/verification/Taiwan_poly.nc
and
GRID_STAT_VERIFICATION_MASK_TEMPLATE = /home/nowcast/tina/verification/Taiwan_poly.nc

In both cases, I get output for my masked domain, but in addition get output for the FULL domain.  I'm hoping to turn that off to save processing time.

Thanks,
Tina

**Comment by georgemccabe** [comment, marked as answer]
Hi @CPKalb,

The default value of mask.grid in GridStat is a list with just the string "FULL"

https://github.com/dtcenter/MET/blob/5e1f551d86fcb5b731e3a83c0b043dbd7ef3c7aa/met/data/config/GridStatConfig_default#L117-L118

Since the default value is not an empty list, the METplus config variable GRID_STAT_MASK_GRID can be set to an empty string which will set the value to an empty string. 

The following variables should accomplish what you need:

```
GRID_STAT_MASK_POLY = /home/nowcast/tina/verification/Taiwan_poly.nc
GRID_STAT_MASK_GRID =
```
